book/src/products/mosaic/fragmentation-in-layer-2.md

```diff
@@ -0,0 +1,17 @@
+# Fragmentation in DeFi's Layer 2
+
+---
+
+While the Ethereum protocol’s revolutionary smart contract functionality is what makes DeFi possible, the ETH transaction speed for DApps is [notoriously slow](https://www.coindesk.com/learn/ethereum-101/will-ethereum-scale), and it stands in the way of further DeFi development and mass adoption. Just as a point of reference, Ethereum’s transaction throughput is about [1,500 times](https://hackernoon.com/the-blockchain-scalability-problem-the-race-for-visa-like-transaction-speed-5cce48f9d44) slower than that of Visa and about [250,000 times](https://www.barrons.com/articles/alibaba-records-25-3-billion-in-singles-day-sales-1510538618) slower than Alipay’s transaction throughput. 
```

Comment:
Though this is correct, the link `notriously slow` is linking a post that got pulled by CoinDesk in December.
Here's a link to the web archive for it:
https://web.archive.org/web/20210816233523/https://www.coindesk.com/learn/ethereum-101/will-ethereum-scale

Reply:
removed this link